ts/theme/variables.ts

```diff
@@ -64,6 +69,9 @@ const customVariables = Object.assign(materialVariables, {
     return this.fontSizeBase * 1.75;
   },
   lineHeight: 24,
+  get lineHeight1(): ThemeSimpleValue {
```

Comment:
perhaps `lineHeight` should be renamed to `baseLineHeight` ? shouldn't `lineHeight1` be bigger than the base line height? (I suppose `lineHeight1` stands for H1) ? - cc @fpersico